docs/Automation.md

```diff
@@ -188,78 +122,22 @@ Sample `C#` code—this is interactive, but yours doesn’t need to be:
             /// </summary>
             static void Main(string[] args)
             {
```

Comment:
>{ [](start = 12, length = 1)

Should we add a try/catch handler to show the AxeWindowsAutomationException type?

Reply:
I'd like to briefly sync on this offline just to make sure I have this right.

Reply:
Added